docs/admin/kubelet-tls-bootstrapping.md

```diff
@@ -130,6 +130,14 @@ rules:
   verbs: ["create"]
 ```
 
+As of 1.8, equivalent roles to the ones listed above are automatically created as part of the default RBAC roles.
+For 1.8 clusters admins are recommended to bind tokens to the following roles instead of creating their own:
+
+* `system:certificates.k8s.io:certificatesigningrequests:io:certificatesigningrequests:nodeclient`
```

Comment:
they're not quite that crazy:
```
system:certificates.k8s.io:certificatesigningrequests:nodeclient
system:certificates.k8s.io:certificatesigningrequests:selfnodeclient
```

Reply:
@luxas ^

Reply:
Sorry should have caught that. Sent https://github.com/kubernetes/kubernetes.github.io/pull/5684